vimrc

```diff
@@ -1,3 +1,6 @@
+" The tests use these settings to ensure consistent results.
+" If you have problems using this plugin, try using these settings.
```

Comment:
If you are using the plugin and it goes wrong then you have settings _in your vimrc_ which are causing a conflict with the plugin. When that happens you should try to use the settings that are in this file.